src/sort-numbers/built-in-sort-rayane/README.md

```diff
@@ -0,0 +1,49 @@
+# Sort Numbers: use a built-in array method
+
+Takes in an array of numbers and returns a new array with the same numbers sorted from smallest to largest.
+
+---
+
+## Strategy
+
+I chose to add a slice method before passing the compare function to the sort method to prevent the original array to be modified.
```

Comment:
as far as I remember, in Strategy we don't need to to mention the name of methods used, it is left for the Implementation part.

Reply:
also, it's not mentioned that all non-finite numbers should be removed from the array (to pass edge cases)